.github/actions/prepare-workspace/action.yml

```diff
@@ -26,27 +21,3 @@ runs:
       id: go-version
       shell: bash
       run: echo "go-version=$(go version | { read _ _ v _; echo ${v#go}; })" >> $GITHUB_OUTPUT
-
-    - name: Go build cache
-      uses: actions/cache@v3
-      with:
-        path: ${{ steps.go-cache-paths.outputs.go-build }}
-        key: ${{ runner.os }}-go-build-${{ steps.go-version.outputs.go-version }}-${{ inputs.cache_key }}-${{ hashFiles('**/go.sum') }}
-        restore-keys: ${{ runner.os }}-go-build-${{ steps.go-version.outputs.go-version }}-${{ inputs.cache_key }}-
-
-    - name: Go mod cache
```

Comment:
Go mod cache should be safe to keep, no?

Also FYI @jentfoo, since you were changing caches recently.

Reply:
@codingllama I'd remove all caches and see how that impacts our builds. From what I see, restoring the cache takes 1:40m to restore and some time (I was not able to find the exact number) to save. I'd assume that fetching code and building is faster, especially since probably 95% of our dependencies are on GH.

Reply:
I was disabling caching as well, except for shared workflow actions and the api.  The changes here look complementary to that work and good to me.

Reply:
Thanks all, sounds good.